### DISCUSSION TITLE
PrivacyTests

### DISCUSSION THREAD

**Discussion by csagan5** [bromite/bromite#1967] [Ideas]
https://privacytests.org/android.html

Repository is at https://github.com/arthuredelstein/privacytests.org (sponsored by BrowserStack apparently)

I have no interest in increasing the "score" of those green marks but rather to understand some of these tests; @uazo mentioning you in case you are curious as well.

**Comment by uazo** [comment]
yes, it was already under my radar ever since https://github.com/bromite/bromite/discussions/1689
if you can do a check on what can be a priority, we can fix it (or we copy it from brave :). 
maybe some results need to be checked because the signals on the caches are strange to me, while on the blob issue he is [right](https://bugs.chromium.org/p/chromium/issues/detail?id=1261328)
it would also be interesting to offer him some other tests.

more than anything else it amazes me how it manages to generate those results, it would serve me precisely to set up the tests at least on the blink side, but I have not yet understood (reading the repo) and I didn't have time to investigate
